docs/en/query_language/alter.md

```diff
@@ -7,46 +7,118 @@ The `ALTER` query is only supported for `*MergeTree` tables, as well as `Merge`a
 Changing the table structure.
 
 ``` sql
-ALTER TABLE [db].name [ON CLUSTER cluster] ADD|DROP|MODIFY COLUMN ...
+ALTER TABLE [db].name [ON CLUSTER cluster] ADD|DROP|CLEAR|COMMENT|MODIFY COLUMN ...
 ```
 
 In the query, specify a list of one or more comma-separated actions.
 Each action is an operation on a column.
 
 The following actions are supported:
 
+- [ADD COLUMN](#alter_add-column) — Adds a new column to the table.
+- [DROP COLUMN](#alter_drop-column) — Deletes the column.
+- [CLEAR COLUMN](#alter_clear-column) — Resets column values.
+- [COMMENT COLUMN](#alter_comment-column) — Adds a text comment to the column.
+- [MODIFY COLUMN](#alter_modify-column) — Changes column's type and/or default expression.
+
+Detailed description of these actions is shown [below](#alter_add-column).
+
+The `ALTER` query lets you create and delete separate elements (columns) in nested data structures, but not whole nested data structures. To add a nested data structure, you can add columns with a name like `name.nested_name` and the type `Array(T)`. A nested data structure is equivalent to multiple array columns with a name that has the same prefix before the dot.
```

Comment:
You start explaining details of very specific usecases before explaining basic usage. This could be a separate "limitations" section.

Reply:
Agree